tests/SignatureVerifierTest.php

```diff
@@ -31,12 +30,17 @@ public function testCreateFromRootMetadata(): void
         $rootMetadata = RootMetadata::createFromJson(file_get_contents($rootJsonPath));
         self::assertInstanceOf(RootMetadata::class, $rootMetadata);
         $rootMetadata->trust();
-        $keyDb = KeyDB::createFromRootMetadata($rootMetadata);
-        self::assertInstanceOf(KeyDB::class, $keyDb);
+
+        $verifier = SignatureVerifier::createFromRootMetadata($rootMetadata);
+        $verifierKeys = new \ReflectionProperty($verifier, 'keys');
+        $verifierKeys->setAccessible(true);
+        $verifierKeys = $verifierKeys->getValue($verifier);
+
         // Get the first key for comparison.
         $keys = $rootMetadata->getKeys();
         $key = reset($keys);
```

Comment:
Why aren't we just looping through all the keys?

Reply:
We could absolutely do that. But this was the existing test coverage from KeyDBTest, so I decided not to alter it (so that you would approve it without asking questions :))